mne/io/edf/edf.py

```diff
@@ -506,6 +506,20 @@ def _get_info(fname, stim_channel, eog, misc, exclude, preload):
     return info, edf_info, orig_units
 
 
+def _parse_prefilter_string(prefiltering: str):
+    """Parses prefilter string from EDF+ and BDF headers for highpass
+       and lowpass filters"""
+    highpass = np.array(
+        [v for hp in [re.findall(r'HP:\s*([0-9]+[.]*[0-9]*)', filt)
+                      for filt in prefiltering] for v in hp]
+    )
+    lowpass = np.array(
+        [v for hp in [re.findall(r'LP:\s*([0-9]+[.]*[0-9]*)', filt)
+                      for filt in prefiltering] for v in hp]
```

Comment:
It seems to me to be more consistent with the rest of the file to keep the REGEXP here, but if anyone really wants them at the top of the file I'll change it.

In terms of adding unit tests, is there currently any test files for io or edf in particular? If not, are we suggesting I create a new file test_edf.py that just tests if this function captures various highpass and lowpass values from strings?

Reply:
I'd keep the regexes here.

Reply:
I'm ambivalent about where the regex lives and you both like them here so let's leave it for now